.circleci/config.yml

```diff
@@ -6,9 +6,13 @@ version: 2
 jobs:
   test_eventgen:
     machine:
-      image: circleci/classic:latest
+      image: circleci/classic:201808-01
```

Comment:
'latest' refers to an image from 2017 w/ python 3.5 installed. Updating to python 3.7.0 since we explicitly support 3.7+